src/logger.js

```diff
@@ -30,7 +30,7 @@ const initKafkaLoggingProducer = (errorCallBack = console.error) => {
   } catch (err) {
     initSuccess = false;
     errorCallBack(`Failed to initialized Kafka producer for logging error: ${err}`);
-    return Promise.reject();
+    return Promise.resolve();
```

Comment:
Why do we want to start the app if the kafka init failed?

Reply:
hmm not sure. do we want to stop and crash the app? If we crash it, it will try to re-establish the connection and restart the app. Does that make more sense?

Reply:
Yeah, I think that would be better.